mesos/register.go

```diff
@@ -2,10 +2,10 @@ package mesos
 
 import (
 	"fmt"
-	"strings"
-	"strconv"
 	"github.com/CiscoCloud/mesos-consul/registry"
 	"github.com/CiscoCloud/mesos-consul/state"
+	"strconv"
```

Comment:
Ok. It's the lack of a blank line between `"strconv"` and `github.com/.../registry`.